code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -108,7 +108,7 @@
 		force = 30
 		w_class = W_CLASS_LARGE
 		sharpness = sharpness_on
-		sharpness_flags = initial(sharpness_flags)
+		sharpness_flags = SHARP_TIP | SHARP_BLADE | INSULATED_EDGE | HOT_EDGE | CHOPWOOD
```

Comment:
What no
Initial is important because it means that the variable is set to what it initially was
In the instance of flags it means you don't have to fix the flags in two separate places, just one